cloud/blockstore/libs/storage/volume/volume_actor_forward.cpp

```diff
@@ -127,7 +133,8 @@ bool TVolumeActor::HandleRequest(
         return false;
     }
 
-    if (partitionRequests.size() == 1) {
+    // Should always forward requests to partitions for DescribeBlocks method.
+    if (partitionRequests.size() == 1 && !IsDescribeBlocksMethod<TMethod>) {
```

Comment:
Else branch expects that request is for multi-partitioned volume, but it is violated when this condition is true: "IsDescribeBlocksMethod<TMethod> && partitionRequests.size() == 1"

Reply:
else branch not expects that request is for multi-partitioned volume

partitionRequests size will be 1 if we needn't forward requests to all partitions 

for example it happens when request block range contains the only one stripe

Reply:
Here we expect that request is for multi-partitioned volume 

https://github.com/ydb-platform/nbs/pull/662/files/90f196098a1ae3150672705993dbe704b621596a#diff-0e8bbc41061139f49336020564a85e877e96b165f4b9b4ad72414f4754d47c5bL159